src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -50,6 +51,8 @@ public SweepStep (bool sweepSymbols = true)
 
 		protected override void Process ()
 		{
+			sweep_debug_support = Context.HasFeatureValue (FeatureSettings.DebuggerIsSupported, false);
```

Comment:
I'm wondering if we should add a new feature for this type of optimization. Only for the linker. And then in SDK default it when the Debugger.IsSupported is off. Just so that there's a way to control it specifically without controlling the rest of the debugger support changes.

Reply:
Yeah, added an extra option to allow control of what will be kept.